### DISCUSSION TITLE
Light randomly turns on by itself without being asked.

### DISCUSSION THREAD

**Discussion by rnodern** [tuya/tuya-home-assistant#944] [Q&A]
I would like some help to identify what is going on. 
I have these string lights "GP-SW060DC1400". They're basically wifi enabled Christmas lights. Randomly they just turn on without being asked. 
Some background: 
- I have 2 houses setup on one Tuya account, & one Google Home Account; (Home A and Home B)
- These lights were originally in Home A but have now moved to Home B. 
- I have ensured that the device has been removed from Home A in Tuya, Google Home and Home Assistant. 
- When in Home A, these lights used to be triggered at a specific time by an automation in Home Assistant. This automation is no longer enabled in Home A Home Assistant. The fact they turn on at random times kind-of rules out a scheduled automation anyway. 
- I've checked the Tuya app to make sure there is no "Schedule" - again, the randomness of time that they get turned on can't be a schedule. 
- The Home Assistant logbook for the device doesn't tell me much. It doesn't indicate a caller. However, i can see when i manually turn them off in the HA UI after they randomly turn on, i can see the turn off was triggered by service light.turn_off
- It doesn't happen every day. I'd say 3 times a week. At different times of day/night. Sometimes i walk past the room they're in, and just get surprised to see that they're on. 
- No other lights, sensors or smart devices in Home A or Home B are affected by this issue. 

Basically, i just want to get to the bottom of what's going on here and identify what is sending a turn_on to these lights. 
Any ideas?
